script/build-wasm

```diff
@@ -135,23 +135,10 @@ $emcc                                            \
   ${web_dir}/binding.c                           \
   -o target/scratch/tree-sitter.js
 
-# Use terser to write a minified version of `tree-sitter.js` into
-# the `lib/binding_web` directory.
-if [[ "$minify_js" == "1" ]]; then
-  if [ ! -d ${web_dir}/node_modules/terser ]; then
-    (
-      cd ${web_dir}
-      npm install
-    )
-  fi
-  ${web_dir}/node_modules/.bin/terser   \
-    --compress                       \
-    --mangle                         \
-    --keep-classnames                \
-    -- target/scratch/tree-sitter.js \
-    > ${web_dir}/tree-sitter.js
-else
-  cp target/scratch/tree-sitter.js ${web_dir}/tree-sitter.js
-fi
-
+mv target/scratch/tree-sitter.js ${web_dir}/tree-sitter.js
 mv target/scratch/tree-sitter.wasm ${web_dir}/tree-sitter.wasm
+if [[ "$debug" == "1" ]]; then
+  mv target/scratch/tree-sitter.wasm.map ${web_dir}/tree-sitter.wasm.map
+  # For source maps
+  rsync -Rr lib ${web_dir}/
```

Comment:
Copying the directory into itself (lib/ into lib/binding_web/lib/) is not great. Also, this also requires (with my compiler on M1 macOS)

```
opt/homebrew/Cellar/emscripten/3.1.59/libexec/
opt/homebrew/Cellar/emscripten/3.1.59/libexec/cache/sysroot/include/atomic_arch.h
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/compiler-rt/emscripten_exception_builtins.c 
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/compiler-rt/emscripten_tempret.s
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/compiler-rt/stack_limits.S
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/compiler-rt/stack_ops.S
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/emscripten_get_heap_size.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/emscripten_memcpy.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/emscripten_syscall_stubs.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/emscripten_time.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/sbrk.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/wasi-helpers.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/pthread/library_pthread_stub.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/pthread/pthread_self_stub.c
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/dImalloc.c
```
and a bunch of files from

```
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/libc/musl/
opt/homebrew/Cellar/emscripten/3.1.59/libexec/system/lib/compiler-rt/lib/builtins/
```

(might contain typos, written with OCR)